### ISSUE TITLE
Docs refactor

### ISSUE THREAD

**Issue by wey-gu** [nebula-contrib/ngbatis#213]
@CorvusYe maybe we could

- Put the raw markdown of graph-cn docs here
- Leave the vitepress code as is in the graph-cn repo(in your gitee page ci, fetch md files from this repo instead)

This will enable en/cn documentation updates in this repo, and it won't fork the two sources of CN docs between here and graph-cn repo.

What do you think?

**Comment by CorvusYe** [comment]
@wey-gu 
Yes, if it can be maintained in only one place, it is undoubtedly the most elegant way. 

But I've been a bit busy lately, I accepted a job at a school, and now it's just the beginning of the school season.
So I need to handle it later or you can try how to make it better.

**Comment by wey-gu** [comment]
> @wey-gu Yes, if it can be maintained in only one place, it is undoubtedly the most elegant way.
> 
> But I've been a bit busy lately, I accepted a job at a school, and now it's just the beginning of the school season. So I need to handle it later or you can try how to make it better.

Wow, congrats @CorvusYe 老师！You are now literally a Teacher now~

For any help that I could possibly provide, let me know via WeChat!

No worries, I can do the CN docs migration, too, also, let's just take our time~~~

**Comment by CorvusYe** [comment]
Thank you, @wey-gu , for your support as always.
